src/ui/components/Library/Team/View/NewTestRuns/TestRunsStats.tsx

```diff
@@ -11,36 +10,16 @@ export function TestRunsStats() {
   const buildFailuresCount = testRuns.filter(r => r.results.counts.failed > 0).length;
   const buildFailureRate = buildFailuresCount / buildsCount;
 
-  const testFailuresCount = testRuns.reduce((acc, r) => acc + r.results.counts.failed, 0);
-  const testsCount = testRuns.reduce(
-    (acc, r) => acc + r.results.counts.passed + r.results.counts.flaky,
-    0
-  );
-  const testFailureRate = testFailuresCount / testsCount;
+  if (!testRuns.length) {
+    return null;
+  }
 
   return (
     <div className="flex flex-col gap-2 text-sm">
-      <Chart />
-      <div className="flex flex-row gap-2">
-        <div className="flex flex-col gap-1 rounded-lg bg-chrome  p-4">
-          <div className="font-bold">Build failure rate</div>
-          <div className="flex flex-row gap-1">
-            <div className={styles.failureRate}>{(buildFailureRate * 100).toFixed(0)}%</div>
-            <div>
-              ({buildFailuresCount}/{buildsCount})
-            </div>
-          </div>
-        </div>
-        <div className="flex flex-row gap-2">
-          <div className="flex flex-col gap-1 rounded-lg bg-chrome  p-4">
-            <div className="font-bold">Test failure rate</div>
-            <div className="flex flex-row gap-1">
-              <div className={styles.failureRate}>{(testFailureRate * 100).toFixed(0)}%</div>
-              <div>
-                ({testFailuresCount}/{testsCount})
-              </div>
-            </div>
-          </div>
+      <div className="flex flex-col overflow-auto rounded-lg bg-chrome p-4">
+        <Chart />
+        <div>
+          Failure rate: {(buildFailureRate * 100).toFixed(2)}% ({buildFailuresCount}/{buildsCount})
```

Comment:
Maybe tuck the counts behind a tooltip? I'm not sure they're important to see at a glance

Reply:
fixed!